src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,47 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({ tabs, onTabSelected, selectedTabId }) => {
+  const checkedSelectedIdTable =
+    tabs.find(e => e.id === selectedTabId)?.id || tabs[0].id;
+
+  return (
+    <>
+      <div data-cy="TabsComponent">
+        <div className="tabs is-boxed">
+          <ul>
+            {tabs.map(tab => {
+              return (
+                <>
+                  <li
+                    key={tab.id}
+                    className={cn({
+                      'is-active': checkedSelectedIdTable === tab.id,
+                    })}
+                    data-cy="Tab"
+                  >
+                    <a
+                      href={`#${tab.id}`}
+                      data-cy="TabLink"
+                      onClick={() => {
+                        if (tab.id !== checkedSelectedIdTable) {
+                          onTabSelected(tab);
+                        }
+                      }}
+                    >
+                      {tab.title}
+                    </a>
+                  </li>
+                </>
+              );
+            })}
+          </ul>
+        </div>
+
+        <div className="block" data-cy="TabContent">
```

Comment:
this here is a very simple component, but probably it should also be another component: what if we wanted to add more features / styles?

Reply:
I agree with your considerations, however the tests are made to inspect the entire 'Tab Component' code and see each of these tags.

If I try to divide more components, the tests show that TabComponent does not have a 'block' tag with data-cy='TabContent' for example.

I noticed that in this case we do not have flexibility for specific changes, such as creating new components, even if the code works, the tests will fail.